index.html

```diff
@@ -763,6 +769,79 @@ <h2>Creating a PR</h2>
+            or use `&lt;name-of-parameter&gt;` if the parameter is required; when a parameter only has some specific
+            values use `&lt;action (action-1/action-2)&gt;`.
+          </li>
+          <li>
+            `run` - This property is the actioner of the command, in this you set what the command will do when its called.
+          </li>
+        </ul>
+        <h2>Creating a message repeater command</h2>
+        <ul>
+          <li>
+            First, go to `./src/commands/general/` and create a new file called `messageRepeater.ts` or as you wish.
+          </li>
+          <li>
+            Then, add the basic structure of our command:<br>
+            <pre><code style="white-space: normal;">
+              import CommandInt from "@Interfaces/CommandInt";<br><br>
+              const messageRepeater: CommandInt = {<br>
+                name: "messageRepeater",<br>
+                description: "Send a message to the current channel using the bot.",<br>
+                parameters: ["&lt;message&gt;: Message to send."],<br>
+                run: async (message) => {<br>
+                  console.log("Message repeater command called!");<br>
+                },<br>
+              };<br><br>
+              export default messageRepeater;
+            </code></pre>
+            Save the file, start the bot (You can use `npm run dev` instead of `npm run build && npm start`) and then
+            in the chat write `{prefix}messageRepeater` and in the console you will see `Message repeater command called!`.            
+          </li>
+          <li>
+            So, to this moment you have created a simple command that shows you a message in the console, but it does not
+            do what we want.<br><br>
+            First, add `const { channel, commandArguments } = message;` instead of the `console.log`, with this new line you will get 
+            the arguments of the command; in other words, the content of the message split by whitespaces and ignoring the
+            first element (The first element is the command name); and the channel of the message.<br><br>
+            Now, add `const msg = commandArguments.join(" ");` to join the arguments with whitespaces to get the
+            parameter that we require (Message).
+          </li>
+          <li>
+            After of get the message, you need send it again to the current channel, to do this you need add this line:
+            `await channel.send(msg);`; now, you can test the command, if you write `{prefix}messageRepeater Hello` the bot
+            will send `Hello` again to the current channel. But what happen if you write `{prefix}messageRepeater`? You will
+            get an error in the console because the bot cannot send an empty message, and how I fix it?.<br><br>
+            It is easier, you need verify if the command arguments have a length before send the message, you must add the
+            lines below before the `await channel.send(msg);`:
```

Comment:
```suggestion
            After you get the message, you need to send it back to the current channel. To do this, add this line:
            `await channel.send(msg);`. Now you can test the command! If you write `{prefix}messageRepeater Hello` in the channel, the bot
            will send `Hello` back to the current channel. But what will happen if you send `{prefix}messageRepeater`? You will
            get an error in the console because the bot cannot send an empty message. So howdo you fix it?
            You need to verify the command arguments have a length before sending the message. Add these
            lines below before the `await channel.send(msg);`:
```
Verbiage, remove `<br>` tags.